pyxrf/view/fit.enaml

```diff
@@ -683,7 +692,26 @@ enamldef FitView(DockItem): fit_view:
                     img_model_adv.data_dict_keys = img_model_adv.data_dict.keys()
                     img_model_rgb.data_dict_keys = []
                     img_model_rgb.data_dict_keys = img_model_rgb.data_dict.keys()
-
+            CheckBox: cb_tiff:
+                text = 'Save to tiff'
+                checkable = True
+                #minimum_size = (20, 20)
+                checked := fit_model.save_tiff
+                clicked ::
+                    if checked:
```

Comment:
Actually, isn't this already being done in the line `checked := fit_model.save_tiff`?